src/Node/Expression/Variable.php

```diff
@@ -24,7 +24,7 @@ class Variable extends Expression {
     public function getName() {
         if (
             $this->name instanceof Token &&
-            $name = substr($this->name->getText($this->getFileContents()), 1)
+            $name = ltrim($this->name->getText($this->getFileContents()), '$')
```

Comment:
`getName` wouldn't work on `$$var`, because of the check above - `name` is a `Variable` node here. You would have to use `$var->name->getName()`.

I made this change for `"${x}"`. `x` is a variable that doesn't have a dollar sign, so I only want to remove a dollar sign if it exists.